cloud/google/machineactuator.go

```diff
@@ -50,7 +54,12 @@ const (
 	ZoneAnnotationKey    = "gcp-zone"
 	NameAnnotationKey    = "gcp-name"
 
```

Comment:
Hmmm, I think you may need to rebase. You are re-adding the uid logic variable that was removed in another change.